app/controllers/supervisor/base_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Supervisor::BaseController < ApplicationController
+
+  before_action :must_be_supervisor
```

Comment:
@marcgreenstock Thanks for being pingable ;-)
What I found (first round). The `:supervise` ability needs a team to pass in. So, if I do: ```can? :supervise, @team``` it would work. BUT we don't have a team_id yet. I think because we only grant access to the supervisor's `user.roles`, not to the team roles. 
https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/app/models/ability.rb#L31
So, my conclusion is we can't use cancancan here to grant access. But we can use it in the views. Does this make sense?